doc/source/reference/offset_frequency.rst

```diff
@@ -938,6 +945,7 @@ Methods
    :toctree: api/
 
     FY5253.apply
+    FY5253.apply_index
```

Comment:
do we need to add these back if we are deprecating?

Reply:
I don't know why, but sphinx started complaining about these not being present. Which is technically correct, since it's a method on those classes. It just raises.

So I think keep for now, and just remove them all when we enforce the deprecation in 2.0